src/mudlet.cpp

```diff
@@ -3478,96 +3534,122 @@ void mudlet::slot_module_manager_destroyed()
 
 // Convenience helper - may aide things if we want to put files in a different
 // place...!
-QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1, const QString& extra2)
+QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1, const QString& extra2, const QString& homePath)
 {
+    QString configDir;
+    if (self()) {
+        configDir = self()->mConfigDir;
+    } else {
+        configDir = mudlet::getQConfig()->value("configPath").toString();
```

Comment:
`value(QStringLiteral("configPath"))`. You can find these warnings by using Qt Creator 4.6.0 and enabling Clazy (I use level 2): http://blog.qt.io/blog/2018/03/28/qt-creator-4-6-0-released/

Reply:
Or just ensure that `QT_NO_CAST_FROM_ASCII` and `QT_NO_CAST_TO_ASCII` are `#define`d but then *every* raw string that is converted to/from a `QString` without a `tr(`....`)`/`QStringLiteral(`...`)`/`QLatin1String(`...`)` wrapper will cause a compilation failure - so you may have difficulty *seeing the wood for the trees* because there are hundreds if not thousands the last time I tried it...! 😆